force_wfmanager/wfmanager_setup_task.py

```diff
@@ -122,6 +127,12 @@ class WfManagerSetupTask(Task):
     #: Review Task
     review_task = Instance(Task)
 
+    #: Setup Task: this will be a self-reference
+    # that allows the TaskActions added by the global task extension
```

Comment:
Nit: you want all of these comments to be `#:` for Sphinx autodoc to pick it up.